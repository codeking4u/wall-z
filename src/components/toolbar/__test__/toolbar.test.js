import renderer from 'react-test-renderer'

import ToolBar from '../toolbar.component'

test('matches snapshot', () => {
    const tree = renderer.create(<ToolBar />).toJSON();
    expect(tree).toMatchSnapshot();
})