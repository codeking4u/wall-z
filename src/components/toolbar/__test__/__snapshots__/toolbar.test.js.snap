// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="toolbar"
>
  <ul
    className="toolbar__list"
  >
    <li
      className="toolbar__items"
      onClick={[Function]}
    >
      <div
        className=""
        style={
          Object {
            "backgroundColor": "#0097e5",
            "borderColor": "#0097e5",
          }
        }
        title="Select wall"
      >
        <svg
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          viewBox="0 0 1024 1024"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M921.088 103.232L584.832 889.024 465.52 544.512 121.328 440.48zM1004.46.769c-6.096 0-13.52 1.728-22.096 5.36L27.708 411.2c-34.383 14.592-36.56 42.704-4.847 62.464l395.296 123.584 129.36 403.264c9.28 15.184 20.496 22.72 31.263 22.72 11.936 0 23.296-9.152 31.04-27.248l408.272-953.728C1029.148 16.368 1022.86.769 1004.46.769z"
          />
        </svg>
      </div>
    </li>
    <li
      className="toolbar__items"
      onClick={[Function]}
    >
      <div
        className=""
        style={
          Object {
            "backgroundColor": "#000",
            "borderColor": "#fff",
          }
        }
        title="Create wall"
      >
        <svg
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          viewBox="0 0 512 512"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M494 18.02l-101 .103V119h101zm-119 .12l-238 .247V119h238zm-256 .266L18 18.51V119h101zM18 137v110h229V137zm247 0v110h229V137zM18 265v110h101V265zm119 0v110h238V265zm256 0v110h101V265zM18 393v100.98l229-.236V393zm247 0v100.727l229-.237V393z"
          />
        </svg>
      </div>
    </li>
  </ul>
</div>
`;
